src/explore-education-statistics-common/src/components/form/rhf/RHFFormFieldCheckboxGroupsMenu.tsx

```diff
@@ -0,0 +1,38 @@
+import DetailsMenu from '@common/components/DetailsMenu';
+import RHFFormFieldCheckboxSearchSubGroups, {
+  RHFFormFieldCheckboxSearchSubGroupsProps,
+} from '@common/components/form/rhf/RHFFormFieldCheckboxSearchSubGroups';
+import RHFFormCheckboxSelectedCount from '@common/components/form/rhf/RHFFormCheckboxSelectedCount';
+import React from 'react';
+import { FieldValues, useFormContext } from 'react-hook-form';
+import get from 'lodash/get';
+
+interface Props<TFormValues extends FieldValues>
+  extends RHFFormFieldCheckboxSearchSubGroupsProps<TFormValues> {
+  legend: string;
+  open?: boolean;
+}
+
+function RHFFormFieldCheckboxGroupsMenu<TFormValues extends FieldValues>(
+  props: Props<TFormValues>,
+) {
+  const { name, legend, open = false } = props;
+
+  const {
+    formState: { errors },
+  } = useFormContext();
+
+  return (
+    <DetailsMenu
+      open={open}
```

Comment:
Just wondering if we should expose `hiddenText` of the `DetailsMenu` for accessibility?

Reply:
Yeah, good idea.